From b841961b0f32ad76d2353dd83b5790cb0cb7a401 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 7 Oct 2024 13:10:42 +0200
Subject: [PATCH] Disable Tauri auto-updater

It wouldn't be able to update itself anyway, as the binary can't
overwrite itself.
---
 src-tauri/tauri.conf.json | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src-tauri/tauri.conf.json b/src-tauri/tauri.conf.json
index 82550ce0dc39..1df44bd5d6e2 100644
--- a/src-tauri/tauri.conf.json
+++ b/src-tauri/tauri.conf.json
@@ -45,6 +45,7 @@
   "identifier": "today.neil.pocket-sync",
   "plugins": {
     "updater": {
+      "active": false,
       "pubkey": "dW50cnVzdGVkIGNvbW1lbnQ6IG1pbmlzaWduIHB1YmxpYyBrZXk6IEVEQzEwQTA4MUMwQTk0OEEKUldTS2xBb2NDQXJCN1VXbjJHZVZhTkpvcDJTTHcxRXFRd3F4S2dMcS9yc1NJZ2xxbFMvZFJWUS8K",
       "endpoints": [
         "https://github.com/neil-morrison44/pocket-sync/releases/latest/download/latest.json"
-- 
2.46.2

